binaries/workercl/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"log"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/scootdev/scoot/workerapi/client"
+)
+
+// Binary to talk to Cloud Worker
+func main() {
+	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
+	transportFactory := thrift.NewTTransportFactory()
+
+	client, err := client.NewClient(transportFactory, protocolFactory)
+	if err != nil {
+		log.Fatal("Cannot initialize Worker CL: ", err)
+	}
+	err = client.Exec()
+	if err != nil {
+		log.Fatal("error running workercl ", err)
+	}
+}
```

Comment:
That shouldn't be there, I almost made the same mistake in my phab review.
I'll delete it.

On Tue, Jul 5, 2016 at 9:30 AM, Dan Bentley <notifications@github.com>
wrote:

> In binaries/workercl/main.go
> <https://github.com/scootdev/scoot/pull/46#discussion_r69594886>:
>
> > +)
> > +
> > +// Binary to talk to Cloud Worker
> > +func main() {
> > +	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
> > +	transportFactory := thrift.NewTTransportFactory()
> > +
> > +	client, err := client.NewClient(transportFactory, protocolFactory)
> > +	if err != nil {
> > +		log.Fatal("Cannot initialize Worker CL: ", err)
> > +	}
> > +	err = client.Exec()
> > +	if err != nil {
> > +		log.Fatal("error running workercl ", err)
> > +	}
> > +}
>
> (Leaving comment here because I can't figure out how to comment on a
> binary file)
>
> What's binaries/workerserver/main?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scootdev/scoot/pull/46/files/391bed1b9417751e8419274fbd053a3eed14f094#r69594886>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABc5CeW71vNQFiFQA23C76BH_t9hNhstks5qSoaVgaJpZM4JEz9N>
> .
>